_layouts/layout.htm

```diff
@@ -2,13 +2,13 @@
 <html class="h-100" lang="en">
     {% include _head.htm %}
     <body>
+        <script src="/assets/js/core/bootstrap-5.1.3.bundle.min.js" type="text/javascript"></script>
```

Comment:
This will block all parsing of the HTML until this script is loaded by putting it at the top. Can still put it earlier in the page just with a `async` attribute to get different behavior

https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup